inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -165,7 +165,7 @@ target_link_libraries(${TARGET_NAME} PRIVATE pugixml::static openvino::itt ${CMA
 
 target_include_directories(${TARGET_NAME} INTERFACE
             $<BUILD_INTERFACE:${PUBLIC_HEADERS_DIR}>
-            $<INSTALL_INTERFACE:${IE_CPACK_IE_DIR}/include>
+            $<INSTALL_INTERFACE:${IE_CPACK_IE_DIR}/include/ie>
```

Comment:
I suppose this change breaks current IRC package structure

Reply:
Yes we are changing the package structure.